src/js/tabs/configuration.js

```diff
@@ -1,31 +1,15 @@
-'use strict';
```

Comment:
Why was this removed?

Reply:
Added (there is no reason I would have removed it)